tests/queries/0_stateless/01287_max_execution_speed.sql

```diff
@@ -1,5 +1,8 @@
 -- Tags: no-fasttest, no-debug, no-tsan, no-msan, no-asan
 
+SET max_rows_to_read=0;
```

Comment:
Future readers will wonder in l. 3-4:
1. since `0` is the default for both settings already, why is this explicitly set to 0?
2. why do we need to set the values at all?

Kindly add code comments, thanks.